src/librustdoc/clean/mod.rs

```diff
@@ -1490,10 +1490,7 @@ impl<'tcx> Clean<Type> for Ty<'tcx> {
                 let path = external_path(cx, did, false, bindings, substs);
                 bounds.insert(
                     0,
-                    PolyTrait {
-                        trait_: ResolvedPath { path, did, is_generic: false },
-                        generic_params: Vec::new(),
-                    },
```

Comment:
This path is for the first trait bound in a `dyn Trait1 + Trait2 + ...` object, so its `Res` should be correct and preserve the `is_generic: false` behavior.